packages/ui/sass/elements/headings.scss

```diff
@@ -44,30 +44,30 @@ h5,
 
 h1,
```

Comment:
@marinagustint The default for these fonts should be the united font size for h1-h3 (just like it was before). The default for h4-h6 should be the font sizes for gotham. Basically, the classes used for these headings should not have changed only the CSS.